src/pages/_middleware.ts

```diff
@@ -112,5 +113,24 @@ export async function middleware(req: NextRequest) {
 
     const res = NextResponse.next();
     logRequestData(pathname, req.method, res.status);
+
+    // @ts-ignore
+    if (req.nextauth && req.nextauth.token) {
```

Comment:
you might be able to do option chaining here  `if (req?.nextauth?.token) { ... }`

Reply:
Thanks, great point! I am actually going to delete this block since I will not be using the cookie.